vignettes/fims-demo.Rmd

```diff
@@ -9,7 +9,13 @@ vignette: >
 
 
 ```{r setup, include=FALSE}
+
 knitr::opts_chunk$set(echo = TRUE)
+withr::local_options(list(
```

Comment:
Can you please add a comment above this line indicating they why of the following code.